### ISSUE TITLE
Cutom order of the tabs & movable using drag and Drop

### ISSUE THREAD

**Issue by Atharv115** [zombieFox/hexagonTab#14]
---
**I'm always frustrated when I have to reorder things after adding a new tab, using those buttons, but can do nothing to move this with the help of drag and drop reorder this would make it work/ function better**
---
**Use Drag & Drop to reorder the Hexagon Tabs**
---
**and also provide with some highlighting guide for the hexagon tab borders to fit in.**
---
>Waiting eagerly for this feature to come to work

**Hope you will look into this ***Zombiefox*** and make our life a lot better**

**Comment by Atharv115** [comment]
@Atharv115 ![bitmoji](https://sdk.bitmoji.com/render/panel/9217e12b-95fc-4b08-ab39-38ebe89875d9-d24f89b5-ba29-4739-bfb0-352799c3c9cd-v1.png?transparent=1&palette=1&width=246)

**Comment by zombieFox** [comment]
Hi @Atharv115 

This is a known pain point in the UX. I've investigated this in the past and with the current grid implementation drag and drop reordering is not likely. The hex grid is made using [CSS Grid](https://developer.mozilla.org/en-US/docs/Web/CSS/grid). To achieve the hexagonal effect a single bookmark item has to **span 6 cells of the grid**. 

Traditional drag and drop reordering requires the DOM nodes being reordered to be set up in a linear fashion fashion. This will not work when a list item is making use of 'grid-column-start', 'grid-row-start' and 'span'. This screen capture demonstrates the issue:

https://user-images.githubusercontent.com/7115017/168682608-4f1e1a5b-2793-450f-9720-8cef461b383b.mp4

The red box shows how a single bookmark spans across 6 cells. I do not know of any other way to get a hexagon with the grid.

**Comment by Atharv115** [comment]
@zombieFox 
Sorry to tell, but I don't have such technical Knowledge about it, I'm a newbie.
Ik u tried to explain it to your fullest, but maybe I'm dumb!
according to ur explaination, it seems as if you would require more time to sort this issue.
Thanks for your time @zombieFox

**Comment by zombieFox** [comment]
> it seems as if you would require more time to sort this issue.

I would like to stress that this may not be possible at all.

**Comment by Atharv115** [comment]
yep fine @zombieFox  m ok with it, btw ur doing a good job, love it
![bitmoji](https://sdk.bitmoji.com/render/panel/eb00c647-02fe-4b42-9d01-5219eeedccc3-d24f89b5-ba29-4739-bfb0-352799c3c9cd-v1.png?transparent=1&palette=1&width=246)
